src/types/index.ts

```diff
@@ -0,0 +1,14 @@
+export type Problem = {
+    http: httpObject,
```

Comment:
It should be optional:

```suggestion
    http?: httpObject,
```